docs/dev/getting_started.md

```diff
@@ -4,7 +4,7 @@
 
 ### Docker Compose
 
-> **TO BE DEFINED**
+[Docker Compose Env developer setup guide](docker_environment.md)
```

Comment:
This link now can point to
https://github.com/ansible/galaxy_ng/tree/master/dev/compose

All other environments are not recommended anymore.

Reply:
`docker_environment.md` is a symlink to that path